packages/project-account-switcher/src/behavior/ProjectAccountSwitcherBehavior.js

```diff
@@ -55,50 +48,50 @@ export default class ProjectAccountSwitcherBehavior extends Component {
     return projects.find(({ id }) => id === projectId);
   }
 
-  handleTargetClick = event => {
-    const { onTargetClick } = this.props;
-
-    if (onTargetClick) {
-      onTargetClick(event);
-    }
-
-    this.setState({
-      open: !this.state.open
-    });
-  };
-
   handleAccountClick = (event, id) => {
```

Comment:
Wondering if it would make sense to combine `handleAccountClick` and `handleProjectClick` they seem to be doing much of the same thing.